src/graphics/program-lib/program-lib.js

```diff
@@ -13,12 +9,7 @@ const programlib = {
     precisionCode: precisionCode,
     skinCode: skinCode,
     tonemapCode: tonemapCode,
-    versionCode: versionCode,
-
-    basic: basic,
-    particle: particle,
-    skybox: skybox,
-    standard: standard
+    versionCode: versionCode
```

Comment:
I think the remainder if this file should move to deprecated.js .. that's where I think it's the only place it is exposed for compatibility reasons.